package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "bookstore-by-adel-new",
+  "version": "1.0.0",
+  "description": "bookstore",
+  "main": "index.ts",
+  "scripts": {
+    "build": "webpack"
+  },
+  "author": "Adel-Kharbout",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.23.6",
+    "@babel/preset-env": "^7.23.6",
+    "@types/node": "^20.10.5",
+    "@types/react": "^18.2.45",
+    "@types/react-dom": "^18.2.18",
+    "@types/stylelint": "^14.0.0",
```

Comment:
WARN deprecated @types/stylelint@14.0.0: This is a stub types definition. stylelint provides its own type definitions, so you do not need this installed.